chef/cookbooks/tempest/recipes/config.rb

```diff
@@ -53,6 +53,7 @@
 
 users = [
           {'name' => tempest_comp_user, 'pass' => tempest_comp_pass, 'role' => 'Member'},
+          {'name' => tempest_comp_user, 'pass' => tempest_comp_pass, 'role' => 'heat_stack_owner'},
```

Comment:
We shouldn't do that when heat is not deployed, though (since the role doesn't exist). We should probably move https://github.com/dirkmueller/barclamp-tempest/blob/master/chef/cookbooks/tempest/recipes/config.rb#L230 a bit earlier and use this variable.